### PR TITLE
Add hips - A Python astronomy package for HiPS

### PR DESCRIPTION
**Pull request by cdeil**
This PR adds a conda-forge recipe for `hips`.
It's the first one I write - can someone please review?

@tboch @adl1995 @mwcraig @bsipocz - Are you OK to to be listed as co-maintainer for this (to be) Astropy-affiliated package?

### CODE REVIEW

File: recipes/hips/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<36]"
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy>=1.11
```
Comment (bsipocz) [inline recipes/hips/meta.yaml]: I think you need a space between the name and version here and below

**Comment by bsipocz** [comment]
@cdeil - sure, feel free to add me to any astro related package

File: recipes/hips/meta.yaml (hunk 1 above)
Comment (cdeil) [reply recipes/hips/meta.yaml]: Done.

File: recipes/hips/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (bsipocz) [inline recipes/hips/meta.yaml]: you need ``--offline`` for packages that use astropy-helpers

**Comment by mwcraig** [comment]
Feel free to include me as a maintainer!

**Comment by cdeil** [comment]
Is this because I put `Pillow` and it should have been `pillow`?
https://circleci.com/gh/conda-forge/staged-recipes/21049?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
I'll try that now (lower-case), but maybe it's some other issue?

**Comment by tboch** [comment]
@cdeil yes, you can add me as co-maintainer

**Comment by adl1995** [comment]
@cdeil I'm fine with being listed as a co-maintainer. Thanks!

**Comment by cdeil** [comment]
CI now passed, I think this is ready to merge.

@bsipocz - thanks for pointing out the solutions to the build fails.

All - thanks for agreeing to co-maintain this package on conda-forge!

**Comment by dougalsutherland** [comment]
Merged. Thanks @cdeil / everyone else! :)

**Pull request merged**